crates/nargo/src/cli/build_cmd.rs

```diff
@@ -47,6 +51,16 @@ pub fn build_from_path<P: AsRef<Path>>(p: P, allow_warnings: bool) -> Result<(),
     Ok(())
 }
 
+fn build_empty_map(abi: &Abi) -> BTreeMap<String, &str> {
```

Comment:
Can use `util::btree_map` here

Reply:
Yeah, I saw this but unfortunately this opens up a cyclic dependency between `noirc_abi` and `noirc_frontend`. We could move those out into a separate `utils` crate though.

Reply:
That would be quite helpful actually - the frontend crate was never a great place for it.

Reply:
Cool, I've made [a separate PR](https://github.com/noir-lang/noir/pull/560) as this falls OOS for this one. We can merge back into here and clean up once that's in.